WaterLog_Backend/Controllers/SegmentEventsController.cs

```diff
@@ -68,16 +68,23 @@ public async Task AddSegmentEvent([FromBody] SegmentEventsEntry value)
             {
                 DateTime tempTime = DateTime.Now;
                 DateTime returnV = new DateTime(tempTime.Year, tempTime.Month, tempTime.Day, i, 0, 0);
```

Comment:
So everything looks good but your naming conventions are making it hard to read. Rename `returnV` and in your for loop you have `ret` maybe rename that too.

Reply:
renamed variables